{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "045c6462",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysql-connector-python\n",
      "  Using cached mysql_connector_python-8.0.28-py2.py3-none-any.whl (342 kB)\n",
      "Collecting protobuf>=3.0.0\n",
      "  Using cached protobuf-3.19.4-py2.py3-none-any.whl (162 kB)\n",
      "Installing collected packages: protobuf, mysql-connector-python\n",
      "Successfully installed mysql-connector-python-8.0.28 protobuf-3.19.4\n"
     ]
    }
   ],
   "source": [
    "!pip install mysql-connector-python"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "64c8b035",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###1번째 처리: 성공\n",
      "###2번째 처리: 성공\n",
      "###3번째 처리: 성공\n",
      "###4번째 처리: 성공\n",
      "###5번째 처리: 성공\n",
      "###6번째 처리: 성공\n",
      "###7번째 처리: 성공\n",
      "###8번째 처리: 성공\n",
      "###9번째 처리: 성공\n",
      "###10번째 처리: 성공\n",
      "###11번째 처리: 성공\n",
      "###12번째 처리: 성공\n",
      "###13번째 처리: 성공\n",
      "###14번째 처리: 성공\n",
      "###15번째 처리: 성공\n",
      "###16번째 처리: 성공\n",
      "###17번째 처리: 성공\n",
      "###18번째 처리: 성공\n",
      "###19번째 처리: 성공\n",
      "###20번째 처리: 성공\n",
      "###21번째 처리: 성공\n",
      "###22번째 처리: 성공\n",
      "###23번째 처리: 성공\n",
      "###24번째 처리: 성공\n",
      "###25번째 처리: 성공\n",
      "###26번째 처리: 성공\n",
      "###27번째 처리: 성공\n",
      "###28번째 처리: 성공\n",
      "###29번째 처리: 성공\n",
      "###30번째 처리: 성공\n",
      "###31번째 처리: 성공\n",
      "###32번째 처리: 성공\n",
      "###33번째 처리: 성공\n",
      "###34번째 처리: 성공\n",
      "###35번째 처리: 성공\n",
      "###36번째 처리: 성공\n",
      "###37번째 처리: 성공\n",
      "###38번째 처리: 성공\n",
      "###39번째 처리: 성공\n",
      "###40번째 처리: 성공\n",
      "###41번째 처리: 성공\n",
      "###42번째 처리: 성공\n",
      "###43번째 처리: 성공\n",
      "###44번째 처리: 성공\n",
      "###45번째 처리: 성공\n",
      "###46번째 처리: 성공\n",
      "###47번째 처리: 성공\n",
      "###48번째 처리: 성공\n",
      "###49번째 처리: 성공\n",
      "###50번째 처리: 성공\n",
      "###51번째 처리: 성공\n",
      "###52번째 처리: 성공\n",
      "###53번째 처리: 성공\n",
      "###54번째 처리: 성공\n",
      "###55번째 처리: 성공\n",
      "###56번째 처리: 성공\n",
      "###57번째 처리: 성공\n",
      "###58번째 처리: 성공\n",
      "###59번째 처리: 성공\n",
      "###60번째 처리: 성공\n",
      "###61번째 처리: 성공\n",
      "###62번째 처리: 성공\n",
      "###63번째 처리: 성공\n",
      "###64번째 처리: 성공\n",
      "###65번째 처리: 성공\n",
      "###66번째 처리: 성공\n",
      "###67번째 처리: 성공\n",
      "###68번째 처리: 성공\n",
      "###69번째 처리: 성공\n",
      "###70번째 처리: 성공\n",
      "###71번째 처리: 성공\n",
      "###72번째 처리: 성공\n",
      "###73번째 처리: 성공\n",
      "###74번째 처리: 성공\n",
      "###75번째 처리: 성공\n",
      "###76번째 처리: 성공\n",
      "###77번째 처리: 성공\n",
      "###78번째 처리: 성공\n",
      "###79번째 처리: 성공\n",
      "###80번째 처리: 성공\n",
      "###81번째 처리: 성공\n",
      "###82번째 처리: 성공\n",
      "###83번째 처리: 성공\n",
      "###84번째 처리: 성공\n",
      "###85번째 처리: 성공\n",
      "###86번째 처리: 성공\n",
      "###87번째 처리: 성공\n",
      "###88번째 처리: 성공\n",
      "###89번째 처리: 성공\n",
      "###90번째 처리: 성공\n",
      "###91번째 처리: 성공\n",
      "###92번째 처리: 성공\n",
      "###93번째 처리: 성공\n",
      "###94번째 처리: 성공\n",
      "###95번째 처리: 성공\n",
      "###96번째 처리: 성공\n",
      "###97번째 처리: 성공\n",
      "###98번째 처리: 성공\n",
      "###99번째 처리: 성공\n",
      "###100번째 처리: 성공\n",
      "###101번째 처리: 성공\n",
      "###102번째 처리: 성공\n",
      "###103번째 처리: 성공\n",
      "###104번째 처리: 성공\n",
      "###105번째 처리: 성공\n",
      "###106번째 처리: 성공\n",
      "###107번째 처리: 성공\n",
      "###108번째 처리: 성공\n",
      "###109번째 처리: 성공\n",
      "###110번째 처리: 성공\n",
      "###111번째 처리: 성공\n",
      "###112번째 처리: 성공\n",
      "###113번째 처리: 성공\n",
      "###114번째 처리: 성공\n",
      "###115번째 처리: 성공\n",
      "###116번째 처리: 성공\n",
      "###117번째 처리: 성공\n",
      "###118번째 처리: 성공\n",
      "###119번째 처리: 성공\n",
      "###120번째 처리: 성공\n",
      "###121번째 처리: 성공\n",
      "###122번째 처리: 성공\n",
      "###123번째 처리: 성공\n",
      "###124번째 처리: 성공\n",
      "###125번째 처리: 성공\n",
      "###126번째 처리: 성공\n",
      "###127번째 처리: 성공\n",
      "###128번째 처리: 성공\n",
      "###129번째 처리: 성공\n",
      "###130번째 처리: 성공\n",
      "###131번째 처리: 성공\n",
      "###132번째 처리: 성공\n",
      "###133번째 처리: 성공\n",
      "###134번째 처리: 성공\n",
      "###135번째 처리: 성공\n",
      "###136번째 처리: 성공\n",
      "###137번째 처리: 성공\n",
      "###138번째 처리: 성공\n",
      "###139번째 처리: 성공\n",
      "###140번째 처리: 성공\n",
      "###141번째 처리: 성공\n",
      "###142번째 처리: 성공\n",
      "###143번째 처리: 성공\n",
      "###144번째 처리: 성공\n",
      "###145번째 처리: 성공\n",
      "###146번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 28.0 seconds\n",
      "###147번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 27.0 seconds\n",
      "###148번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 25.0 seconds\n",
      "###149번째 처리: 성공\n",
      "###150번째 처리: 성공\n",
      "###151번째 처리: 성공\n",
      "###152번째 처리: 성공\n",
      "###153번째 처리: 성공\n",
      "###154번째 처리: 성공\n",
      "###155번째 처리: 성공\n",
      "###156번째 처리: 성공\n",
      "###157번째 처리: 성공\n",
      "###158번째 처리: 성공\n",
      "###159번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 24.0 seconds\n",
      "###160번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 23.0 seconds\n",
      "###161번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 22.0 seconds\n",
      "###162번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 20.0 seconds\n",
      "###163번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 19.0 seconds\n",
      "###164번째 처리: 성공\n",
      "###165번째 처리: 성공\n",
      "###166번째 처리: 성공\n",
      "###167번째 처리: 성공\n",
      "###168번째 처리: 성공\n",
      "###169번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 18.0 seconds\n",
      "###170번째 처리: 성공\n",
      "###171번째 처리: 성공\n",
      "###172번째 처리: 성공\n",
      "###173번째 처리: 성공\n",
      "###174번째 처리: 성공\n",
      "###175번째 처리: 성공\n",
      "###176번째 처리: 성공\n",
      "###177번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 16.0 seconds\n",
      "###178번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 15.0 seconds\n",
      "###179번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 14.0 seconds\n",
      "###180번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 12.0 seconds\n",
      "###181번째 처리: 성공\n",
      "###182번째 처리: 성공\n",
      "###183번째 처리: 성공\n",
      "###184번째 처리: 성공\n",
      "###185번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 11.0 seconds\n",
      "###186번째 처리: 성공\n",
      "###187번째 처리: 성공\n",
      "###188번째 처리: 성공\n",
      "###189번째 처리: 성공\n",
      "###190번째 처리: 성공\n",
      "###191번째 처리: 성공\n",
      "###192번째 처리: 성공\n",
      "###193번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 10.0 seconds\n",
      "###194번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Timed out\n",
      "###195번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Timed out\n",
      "###196번째 처리: 성공\n",
      "###197번째 처리: 성공\n",
      "###198번째 처리: 성공\n",
      "###199번째 처리: 성공\n",
      "###200번째 처리: 성공\n",
      "###201번째 처리: 성공\n",
      "###202번째 처리: 성공\n",
      "###203번째 처리: 성공\n",
      "###204번째 처리: 성공\n",
      "###205번째 처리: 성공\n",
      "###206번째 처리: 성공\n",
      "###207번째 처리: 성공\n",
      "###208번째 처리: 성공\n",
      "###209번째 처리: 성공\n",
      "###210번째 처리: 성공\n",
      "###211번째 처리: 성공\n",
      "###212번째 처리: 성공\n",
      "###213번째 처리: 성공\n",
      "###214번째 처리: 성공\n",
      "###215번째 처리: 성공\n",
      "###216번째 처리: 성공\n",
      "###217번째 처리: 성공\n",
      "###218번째 처리: 성공\n",
      "###219번째 처리: 성공\n",
      "###220번째 처리: 성공\n",
      "###221번째 처리: 성공\n",
      "###222번째 처리: 성공\n",
      "###223번째 처리: 성공\n",
      "###224번째 처리: 성공\n",
      "###225번째 처리: 성공\n",
      "###226번째 처리: 성공\n",
      "###227번째 처리: 성공\n",
      "###228번째 처리: 성공\n",
      "###229번째 처리: 성공\n",
      "###230번째 처리: 성공\n",
      "###231번째 처리: 성공\n",
      "###232번째 처리: 성공\n",
      "###233번째 처리: 성공\n",
      "###234번째 처리: 성공\n",
      "###235번째 처리: 성공\n",
      "###236번째 처리: 성공\n",
      "###237번째 처리: 성공\n",
      "###238번째 처리: 성공\n",
      "###239번째 처리: 성공\n",
      "###240번째 처리: 성공\n",
      "###241번째 처리: 성공\n",
      "###242번째 처리: 성공\n",
      "###243번째 처리: 성공\n",
      "###244번째 처리: 성공\n",
      "###245번째 처리: 성공\n",
      "###246번째 처리: 성공\n",
      "###247번째 처리: 성공\n",
      "###248번째 처리: 성공\n",
      "###249번째 처리: 성공\n",
      "###250번째 처리: 성공\n",
      "###251번째 처리: 성공\n",
      "###252번째 처리: 성공\n",
      "###253번째 처리: 성공\n",
      "###254번째 처리: 성공\n",
      "###255번째 처리: 성공\n",
      "###256번째 처리: 성공\n",
      "###257번째 처리: 성공\n",
      "###258번째 처리: 성공\n",
      "###259번째 처리: 성공\n",
      "###260번째 처리: 성공\n",
      "###261번째 처리: 성공\n",
      "###262번째 처리: 성공\n",
      "###263번째 처리: 성공\n",
      "###264번째 처리: 성공\n",
      "###265번째 처리: 성공\n",
      "###266번째 처리: 성공\n",
      "###267번째 처리: 성공\n",
      "###268번째 처리: 성공\n",
      "###269번째 처리: 성공\n",
      "###270번째 처리: 성공\n",
      "###271번째 처리: 성공\n",
      "###272번째 처리: 성공\n",
      "###273번째 처리: 성공\n",
      "###274번째 처리: 성공\n",
      "###275번째 처리: 성공\n",
      "###276번째 처리: 성공\n",
      "###277번째 처리: 성공\n",
      "###278번째 처리: 성공\n",
      "###279번째 처리: 성공\n",
      "###280번째 처리: 성공\n",
      "###281번째 처리: 성공\n",
      "###282번째 처리: 성공\n",
      "###283번째 처리: 성공\n",
      "###284번째 처리: 성공\n",
      "###285번째 처리: 성공\n",
      "###286번째 처리: 성공\n",
      "###287번째 처리: 성공\n",
      "###288번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 34.0 seconds\n",
      "###289번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 33.0 seconds\n",
      "###290번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 31.0 seconds\n",
      "###291번째 처리: 성공\n",
      "###292번째 처리: 성공\n",
      "###293번째 처리: 성공\n",
      "###294번째 처리: 성공\n",
      "###295번째 처리: 성공\n",
      "###296번째 처리: 성공\n",
      "###297번째 처리: 성공\n",
      "###298번째 처리: 성공\n",
      "###299번째 처리: 성공\n",
      "###300번째 처리: 성공\n",
      "###301번째 처리: 성공\n",
      "###302번째 처리: 성공\n",
      "###303번째 처리: 성공\n",
      "###304번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 30.0 seconds\n",
      "###305번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 29.0 seconds\n",
      "###306번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 27.0 seconds\n",
      "###307번째 처리: 성공\n",
      "###308번째 처리: 성공\n",
      "###309번째 처리: 성공\n",
      "###310번째 처리: 성공\n",
      "###311번째 처리: 성공\n",
      "###312번째 처리: 성공\n",
      "###313번째 처리: 성공\n",
      "###314번째 처리: 성공\n",
      "###315번째 처리: 성공\n",
      "###316번째 처리: 성공\n",
      "###317번째 처리: 성공\n",
      "###318번째 처리: 성공\n",
      "###319번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 26.0 seconds\n",
      "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c5fa00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e1c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdfe20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdfd90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844efa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef419d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef414f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084829a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c5fa00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844eb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdfe20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844efa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdfca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084826d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107490e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085a09a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f49a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c5fa00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ecd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10803f610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10803fe20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844eb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105f89c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084940a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10803fb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10803f070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084828e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef41a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef414f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4a610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4ae20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e1c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4aa30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4a370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084829d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ed00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084822e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084947f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084941f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ed00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4be20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4ba00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b1f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084825e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844eaf0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084940a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef439a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef434f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4be20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b9a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084947c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088404c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084827f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084823a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108841610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108841e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084948e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084941f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108841a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088414f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084944f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844edc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108843610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108843e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ea90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088439a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088434f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084947c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844dfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108840280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084829a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844efa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b5e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4be20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bf10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef473a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef475e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4be80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107853e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef476a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1061424c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105ea59a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1061424c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef47ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef466d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1061425b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e65dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bc7160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e28ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084824f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106142fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084948e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bc7340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105ea5880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef46e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084944c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084821f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884a730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c513a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884a5e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884af40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084944c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1061424c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084824c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884a6a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c513a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084940a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106ce4190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884aeb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084946d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107853370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b8e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108426970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084823d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084944c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ea90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bd60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844dfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bd60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084823a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bc40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084941f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884fe80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106ce43a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e53460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bd60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884fb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088486d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bcd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106142460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106f3b640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108848880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108848ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084940a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084946d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884c730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084823a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b5e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084947f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884c8e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884cf40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106f3b640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084946d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108853790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c7e400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108853940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108853fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4bd60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10884f460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a47f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c0b5e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085a02b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108853ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a44f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ea90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bc7340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef4b5b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e8f4c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084948e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bb670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bb820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bbe80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c84c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bbb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084823d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107490e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107853760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdf880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bdfee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bc7100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084827f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bbb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef44730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084941c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107bc7160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef448e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef44f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bb3a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084824c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084949d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084822e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bb3a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088477f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108857040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084940a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084824c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078f4a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088474f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088478e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c0b5e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef438e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef44d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef442e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088475b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef44460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105ea57f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e28ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088471f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef430a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106cca490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084824f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10844e8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108494f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bbd00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1073985b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088473a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bb850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084820a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105e53460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085bbc10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107c51520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108482f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108847670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10ef43fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "###865번째 처리: 성공\n",
      "###866번째 처리: 성공\n",
      "###867번째 처리: 성공\n",
      "###868번째 처리: 성공\n",
      "###869번째 처리: 성공\n",
      "###870번째 처리: 성공\n",
      "###871번째 처리: 성공\n",
      "###872번째 처리: 성공\n",
      "###873번째 처리: 성공\n",
      "###874번째 처리: 성공\n",
      "###875번째 처리: 성공\n",
      "###876번째 처리: 성공\n",
      "###877번째 처리: 성공\n",
      "###878번째 처리: 성공\n",
      "###879번째 처리: 성공\n",
      "###880번째 처리: 성공\n",
      "###881번째 처리: 성공\n",
      "###882번째 처리: 성공\n",
      "###883번째 처리: 성공\n",
      "###884번째 처리: 성공\n",
      "###885번째 처리: 성공\n",
      "###886번째 처리: 성공\n",
      "###887번째 처리: 성공\n",
      "###888번째 처리: 성공\n",
      "###889번째 처리: 성공\n",
      "###890번째 처리: 성공\n",
      "###891번째 처리: 성공\n",
      "###892번째 처리: 성공\n",
      "###893번째 처리: 성공\n",
      "###894번째 처리: 성공\n",
      "###895번째 처리: 성공\n",
      "###896번째 처리: 성공\n",
      "###897번째 처리: 성공\n",
      "###898번째 처리: 성공\n",
      "###899번째 처리: 성공\n",
      "###900번째 처리: 성공\n",
      "###901번째 처리: 성공\n",
      "###902번째 처리: 성공\n",
      "###903번째 처리: 성공\n",
      "###904번째 처리: 성공\n",
      "###905번째 처리: 성공\n",
      "###906번째 처리: 성공\n",
      "###907번째 처리: 성공\n",
      "###908번째 처리: 성공\n",
      "###909번째 처리: 성공\n",
      "###910번째 처리: 성공\n",
      "###911번째 처리: 성공\n",
      "###912번째 처리: 성공\n",
      "###913번째 처리: 성공\n",
      "###914번째 처리: 성공\n",
      "###915번째 처리: 성공\n",
      "###916번째 처리: 성공\n",
      "###917번째 처리: 성공\n",
      "###918번째 처리: 성공\n",
      "###919번째 처리: 성공\n",
      "###920번째 처리: 성공\n",
      "###921번째 처리: 성공\n",
      "###922번째 처리: 성공\n",
      "###923번째 처리: 성공\n",
      "###924번째 처리: 성공\n",
      "###925번째 처리: 성공\n",
      "###926번째 처리: 성공\n",
      "###927번째 처리: 성공\n",
      "###928번째 처리: 성공\n",
      "###929번째 처리: 성공\n",
      "###930번째 처리: 성공\n",
      "###931번째 처리: 성공\n",
      "###932번째 처리: 성공\n",
      "###933번째 처리: 성공\n",
      "###934번째 처리: 성공\n",
      "###935번째 처리: 성공\n",
      "###936번째 처리: 성공\n",
      "###937번째 처리: 성공\n",
      "###938번째 처리: 성공\n",
      "###939번째 처리: 성공\n",
      "###940번째 처리: 성공\n",
      "###941번째 처리: 성공\n",
      "###942번째 처리: 성공\n",
      "###943번째 처리: 성공\n",
      "###944번째 처리: 성공\n",
      "###945번째 처리: 성공\n",
      "###946번째 처리: 성공\n",
      "###947번째 처리: 성공\n",
      "###948번째 처리: 성공\n",
      "###949번째 처리: 성공\n",
      "###950번째 처리: 성공\n",
      "###951번째 처리: 성공\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###952번째 처리: 성공\n",
      "###953번째 처리: 성공\n",
      "###954번째 처리: 성공\n",
      "###955번째 처리: 성공\n",
      "###956번째 처리: 성공\n",
      "###957번째 처리: 성공\n",
      "###958번째 처리: 성공\n",
      "###959번째 처리: 성공\n",
      "###960번째 처리: 성공\n",
      "###961번째 처리: 성공\n",
      "###962번째 처리: 성공\n",
      "###963번째 처리: 성공\n",
      "###964번째 처리: 성공\n",
      "###965번째 처리: 성공\n",
      "###966번째 처리: 성공\n",
      "###967번째 처리: 성공\n",
      "###968번째 처리: 성공\n",
      "###969번째 처리: 성공\n",
      "###970번째 처리: 성공\n",
      "###971번째 처리: 성공\n",
      "###972번째 처리: 성공\n",
      "###973번째 처리: 성공\n",
      "###974번째 처리: 성공\n",
      "###975번째 처리: 성공\n",
      "###976번째 처리: 성공\n",
      "###977번째 처리: 성공\n",
      "###978번째 처리: 성공\n",
      "###979번째 처리: 성공\n",
      "###980번째 처리: 성공\n",
      "###981번째 처리: 성공\n",
      "###982번째 처리: 성공\n",
      "###983번째 처리: 성공\n",
      "###984번째 처리: 성공\n",
      "###985번째 처리: 성공\n",
      "###986번째 처리: 성공\n",
      "###987번째 처리: 성공\n",
      "###988번째 처리: 성공\n",
      "###989번째 처리: 성공\n",
      "###990번째 처리: 성공\n",
      "###991번째 처리: 성공\n",
      "###992번째 처리: 성공\n",
      "###993번째 처리: 성공\n",
      "###994번째 처리: 성공\n",
      "###995번째 처리: 성공\n",
      "###996번째 처리: 성공\n",
      "###997번째 처리: 성공\n",
      "###998번째 처리: 성공\n",
      "###999번째 처리: 성공\n",
      "###1000번째 처리: 성공\n",
      "###1001번째 처리: 성공\n",
      "###1002번째 처리: 성공\n",
      "###1003번째 처리: 성공\n",
      "###1004번째 처리: 성공\n",
      "###1005번째 처리: 성공\n",
      "###1006번째 처리: 성공\n",
      "###1007번째 처리: 성공\n",
      "###1008번째 처리: 성공\n",
      "###1009번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 15.0 seconds\n",
      "###1010번째 처리: 성공\n",
      "###1011번째 처리: 성공\n",
      "###1012번째 처리: 성공\n",
      "###1013번째 처리: 성공\n",
      "###1014번째 처리: 성공\n",
      "###1015번째 처리: 성공\n",
      "###1016번째 처리: 성공\n",
      "###1017번째 처리: 성공\n",
      "###1018번째 처리: 성공\n",
      "###1019번째 처리: 성공\n",
      "###1020번째 처리: 성공\n",
      "###1021번째 처리: 성공\n",
      "###1022번째 처리: 성공\n",
      "###1023번째 처리: 성공\n",
      "###1024번째 처리: 성공\n",
      "###1025번째 처리: 성공\n",
      "###1026번째 처리: 성공\n",
      "###1027번째 처리: 성공\n",
      "###1028번째 처리: 성공\n",
      "###1029번째 처리: 성공\n",
      "###1030번째 처리: 성공\n",
      "###1031번째 처리: 성공\n",
      "###1032번째 처리: 성공\n",
      "###1033번째 처리: 성공\n",
      "###1034번째 처리: 성공\n",
      "###1035번째 처리: 성공\n",
      "###1036번째 처리: 성공\n",
      "###1037번째 처리: 성공\n",
      "###1038번째 처리: 성공\n",
      "###1039번째 처리: 성공\n",
      "###1040번째 처리: 성공\n",
      "###1041번째 처리: 성공\n",
      "###1042번째 처리: 성공\n",
      "###1043번째 처리: 성공\n",
      "###1044번째 처리: 성공\n",
      "###1045번째 처리: 성공\n",
      "###1046번째 처리: 성공\n",
      "###1047번째 처리: 성공\n",
      "###1048번째 처리: 성공\n",
      "###1049번째 처리: 성공\n",
      "###1050번째 처리: 성공\n",
      "###1051번째 처리: 성공\n",
      "###1052번째 처리: 성공\n",
      "###1053번째 처리: 성공\n",
      "###1054번째 처리: 성공\n",
      "###1055번째 처리: 성공\n",
      "###1056번째 처리: 성공\n",
      "###1057번째 처리: 성공\n",
      "###1058번째 처리: 성공\n",
      "###1059번째 처리: 성공\n",
      "###1060번째 처리: 성공\n",
      "###1061번째 처리: 성공\n",
      "###1062번째 처리: 성공\n",
      "###1063번째 처리: 성공\n",
      "###1064번째 처리: 성공\n",
      "###1065번째 처리: 성공\n",
      "###1066번째 처리: 성공\n",
      "###1067번째 처리: 성공\n",
      "###1068번째 처리: 성공\n",
      "###1069번째 처리: 성공\n",
      "###1070번째 처리: 성공\n",
      "###1071번째 처리: 성공\n",
      "###1072번째 처리: 성공\n",
      "###1073번째 처리: 성공\n",
      "###1074번째 처리: 성공\n",
      "###1075번째 처리: 성공\n",
      "###1076번째 처리: 성공\n",
      "###1077번째 처리: 성공\n",
      "###1078번째 처리: 성공\n",
      "###1079번째 처리: 성공\n",
      "###1080번째 처리: 성공\n",
      "###1081번째 처리: 성공\n",
      "###1082번째 처리: 성공\n",
      "###1083번째 처리: 성공\n",
      "###1084번째 처리: 성공\n",
      "###1085번째 처리: 성공\n",
      "###1086번째 처리: 성공\n",
      "###1087번째 처리: 성공\n",
      "###1088번째 처리: 성공\n",
      "###1089번째 처리: 성공\n",
      "###1090번째 처리: 성공\n",
      "###1091번째 처리: 성공\n",
      "###1092번째 처리: 성공\n",
      "###1093번째 처리: 성공\n",
      "###1094번째 처리: 성공\n",
      "###1095번째 처리: 성공\n",
      "###1096번째 처리: 성공\n",
      "###1097번째 처리: 성공\n",
      "###1098번째 처리: 성공\n",
      "###1099번째 처리: 성공\n",
      "###1100번째 처리: 성공\n",
      "###1101번째 처리: 성공\n",
      "###1102번째 처리: 성공\n",
      "###1103번째 처리: 성공\n",
      "###1104번째 처리: 성공\n",
      "###1105번째 처리: 성공\n",
      "###1106번째 처리: 성공\n",
      "###1107번째 처리: 성공\n",
      "###1108번째 처리: 성공\n",
      "###1109번째 처리: 성공\n",
      "###1110번째 처리: 성공\n",
      "###1111번째 처리: 성공\n",
      "###1112번째 처리: 성공\n",
      "###1113번째 처리: 성공\n",
      "###1114번째 처리: 성공\n",
      "###1115번째 처리: 성공\n",
      "###1116번째 처리: 성공\n",
      "###1117번째 처리: 성공\n",
      "###1118번째 처리: 성공\n",
      "###1119번째 처리: 성공\n"
     ]
    }
   ],
   "source": [
    "import mysql.connector\n",
    "import requests, json\n",
    "import time\n",
    "\n",
    "# 데이터베이스 컨넥션을 생성한다.\n",
    "mydb = mysql.connector.connect(\n",
    "    host=\"127.0.0.1\",\n",
    "    user=\"smartnqi\",\n",
    "    passwd=\"nwai1234!\",\n",
    "    database=\"smart\"\n",
    ")\n",
    "\n",
    "# 커서를 생성한다.\n",
    "cur = mydb.cursor()\n",
    "\n",
    "# 데이터를 조회할 쿼리문을 작성한다. \n",
    "sql = '''\n",
    "    select \n",
    "        'call' as dataType,\n",
    "        phone_no,\n",
    "        meastime,\n",
    "        networkId,\n",
    "        groupId,\n",
    "        currentTime,\n",
    "        timeline,\n",
    "        cellId,\n",
    "        currentCount,\n",
    "        ispId,\n",
    "        testNetworkType,\n",
    "        userInfo1,\n",
    "        userInfo2,\n",
    "        siDo,\n",
    "        guGun,\n",
    "        addressDetail,\n",
    "        udpJitter,\n",
    "        downloadBandwidth,\n",
    "        uploadBandwidth,\n",
    "        sinr,\n",
    "        isWifi,\n",
    "        latitude,\n",
    "        longitude,\n",
    "        bandType,\n",
    "        p_dl_earfcn,\n",
    "        p_pci,\n",
    "        p_rsrp,\n",
    "        NR_EARFCN,\n",
    "        NR_PCI,\n",
    "        NR_RSRP,\n",
    "        NR_SINR\n",
    "        from tb_ndm_data_measure  \n",
    "        where \n",
    "            meastime between 20211101000000000 and 20211101235959999\n",
    "            and userInfo1 = '서울특별시-금천구-가산동'\n",
    "            \n",
    "        order by savetime\n",
    "    '''\n",
    "\n",
    "# 쿼리문을 실행한다.\n",
    "cur.execute(sql)\n",
    "row_headers=[x[0] for x in cur.description]\n",
    "res = cur.fetchall()\n",
    "\n",
    "# 데이터를 가져와서 JSON 데이터를 생성한다.\n",
    "for idx, row in enumerate(res):\n",
    "    tmpData = dict(zip(row_headers,row))\n",
    "    jsonData = json.dumps(tmpData)\n",
    "    try:\n",
    "        r = requests.post(\"http://localhost:8000/monitor/json/\", data=jsonData)\n",
    "        # 전송실패 시 오류값을 출력한다.\n",
    "        if r.status_code == 200:\n",
    "            print(f\"###{idx+1}번째 처리: 성공\")\n",
    "        else:\n",
    "            print(f\"###{idx+1}번째 처리: 오류{r.status_code}{r.text}\")\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(str(e))\n",
    "\n",
    "#     # 초당 3개씩 전달하도록 한다. \n",
    "#     # 너무 다량의 데이터를 연속해서 보내면 서버단에서 부하로 오류가 발생한다. \n",
    "#     if idx % 3 == 0:\n",
    "#         time.sleep(1)\n",
    "                  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "92cb7d88",
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
